Kitodo/src/main/java/org/kitodo/production/services/data/TaskService.java

```diff
@@ -969,4 +913,19 @@ public static int getCorrespondingTemplateTaskId(Task task) {
         }
         return -1;
     }
+
+    // === alternative functions that are no longer required ===
+
+    /**
+     * Find object in ES and convert it to Interface.
+     *
+     * @param id
+     *            object id
+     * @return Interface object
+     * @deprecated Use {@link #getById(Integer)}.
+     */
+    @Deprecated
+    public Task findById(Integer id) throws DAOException {
```

Comment:
Why is a new method already marked as deprecated instead of replacing it old usages?